src/inference/graph_inference.cpp

```diff
@@ -17,11 +17,13 @@
 #include "graph_inference.h"
 
 #include <stdio.h>
+#include <math.h>
 #include <queue>
 #include <unordered_set>
 
 #include "stringprintf.h"
 #include "glog/logging.h"
+#include "gflags/gflags.h"
```

Comment:
On my machine it wasn't compiling without that include. I will double check to be sure that it is necessary.